### ISSUE TITLE
Error: Invariant violation: View config not found for name RCTVLCPlayer

### ISSUE THREAD

**Issue by rsispal** [kikill95/react-native-vlc-player#15]
Hi,

Struggling to get going with this library. Followed the instructions and everything seemed to have installed correctly.

Though, I receive error `Invariant violation: View config not found for name RCTVLCPlayer`.

Is anybody who has had any success with this or the iOS project by @xiongchuan86 point me in the right direction? Perhaps even try with the package versions below?

**Package Versions:**
```
"dependencies": {
    "react": "^16.4.0",
    "react-native": "^0.55.4",
    "react-native-vlc-player": "git+https://github.com/kikill95/react-native-vlc-player.git",
  },
```

**Comment by kikill95** [comment]
Hello, @rsispal and thank you for your feedback

Could you please provide some react code?

Also, we know about a lot of issues and working actively on full redeveloping, that's why we aren't on npm, yet

**Comment by rsispal** [comment]
Absolutely. I am going to try once more today/tomorrow just to ensure I followed the instructions correctly. Will report back with some code and an example app to help demonstrate the error if it still occurs.

**Comment by kikill95** [comment]
Great, @rsispal

**Comment by joaodematejr** [comment]
+1
